src/components/Fabrica.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './Fabrica.css';
+
+export const Celda = (props) => { 
+  return (
+    <div id={"celda-"+props.id } className="celda">
+      { props.id }
+    </div>
+  )
+} 
+
+const crearGrilla = () => {
+  let rows = [];
```

Comment:
Esto lo podrías hacer un poco menos "nasty" con ES6:

```
return Array.from(Array(cantidadDeCeldas)).map((_, index) => <Celda id={index} key={index}/>);
```

Reply:
Eso no es exactamente lo mismo que hay hoy, o si? `crearGrilla()` genera una lista de `{id: i}` y luego la mapea a componentes `Celda`, el día de mañana podriamos extender eso a un objeto con un poco más de comportamiento. 
¿Como sería el equivalente con esa sintaxis?

Reply:
Almost the same: 
- `Array.from(Array(cantidadDeCeldas))` - Retorna una lista de tantos elementos como cantidadDeCeldas diga.

```
const list = Array.from(Array(3))
> [undefined, undefined, undefined]
list.length
> 3
```

- `.map((_, index) => <Celda id={index} key={index}/>);` - Bastante self-explanatory, te mapea la lista a Celdas con el indice (empezando de 0).